packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -4125,255 +4041,95 @@ class RenderSemanticsAnnotations extends RenderProxyBox {
 
   /// If non-null, sets the [SemanticsFlag.hasCheckedState] semantic to true and
   /// the [SemanticsConfiguration.isChecked] semantic to the given value.
-  bool? get checked => _checked;
-  bool? _checked;
-  set checked(bool? value) {
-    if (checked == value)
-      return;
-    _checked = value;
-    markNeedsSemanticsUpdate();
-  }
+  bool? get checked => _properties.checked;
```

Comment:
I would be in favor of removing these getters as well, if that's not breaking anything. Are these actually used anywhere?

Reply:
They are only used locally in the framework afaik.  They are probably used in tests.  There is tons of documentation written for them.  I was worried that some of that information might only live there.  It seems like SemanticsProperties is the proper place for that documentation though.

Reply:
+1. If this information is not also available on SemanticsProperties let's move it there and get rid of these here.